components/script/docs/JS-Servos-only-GC.md

```diff
@@ -164,8 +164,8 @@ relationship. The `Document` just has a pointer to a `Window`, one of many
 pointers to that object, which can live in native DOM data structures or in
 JavaScript objects. These are precisely the pointers we need to tell the
 garbage collector about. We do this with a
-[custom type for traced pointers: `JS<T>`][js] (for example, the `JS<Window>`
-above). The implementation of `trace` for `JS<T>` is not auto-generated; this
+[custom type for traced pointers: `Dom<T>`][js] (for example, the `Dom<Window>`
```

Comment:
nit: Maybe change the link name too? (right now it's called `js`).

Reply:
Done.